### PR TITLE
new catalog metadata column `MimeType` (dexterity and at) https://git…

### PR DESCRIPTION
**Pull request by fgrcon**
…hub.com/plone/Products.CMFPlone/issues/1995

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ New features:
 - Update mockup to latest version.
   [thet]
 
+- new metadata catalog columń MimeType
```
Comment (davisagli) [inline CHANGES.rst]: typo -- there's an accent on the n

File: Products/CMFPlone/CatalogTool.py [hunk 2]
```diff
@@ -285,6 +285,16 @@ def getIcon(obj):
 
 
 @indexer(Interface)
+def MimeType(obj):
+    if obj.portal_type in ['File','Image']:
```
Comment (davisagli) [inline Products/CMFPlone/CatalogTool.py]: Why limit it to File and Image? Custom types could have a file or image field too. I would check to see if the object has a primary field, and if so use the mimetype from that field's value.

File: Products/CMFPlone/profiles/default/catalog.xml [hunk 3]
```diff
@@ -135,4 +135,5 @@
  <column value="portal_type"/>
  <column value="review_state"/>
  <column value="start"/>
+ <column value="MimeType"/>
```
Comment (davisagli) [inline Products/CMFPlone/profiles/default/catalog.xml]: I know we don't really have a convention for whether to use camel case or underscores for catalog metadata columns. But the only ones that are capitalized are named after CMF accessor methods. I would probably call this `mime_type`

**Comment by fgrcon** [comment]
@davisagli  @jensens :   thanks for reviewing  ....

**Comment by fgrcon** [comment]
@jensens  thanks for reviews, done  http://jenkins.plone.org/job/pull-request-5.1/1487

**Pull request merged**